plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -65,7 +65,7 @@ const PodContainerConfigSection = ({containerConfig, appConfig, onEditClick, ind
       </ConfigurationMapRow>
 
       {/* Resources */}
-      {fields.resources.cpus && (
+      {!!fields.resources.cpus && (
```

Comment:
@mlunoe thanks! However I was going to go with something that satisfies both @nLight and @orlandohohmeier and do something like:

```js
function isValueVisible(value) {
   return (isNumber(value) && (parseInt(value) !== 0) || isDefined(value);
}

...

      {/* Resources */}
      {isValueVisible(fields.resources.cpus) && (
        <ConfigurationMapRow>
          <ConfigurationMapLabel>CPUs</ConfigurationMapLabel>
          <ConfigurationMapValue value={fields.resources.cpus} />
          <ConfigurationMapEditAction
```